pfcpiface/p4rt_translator_test.go

```diff
@@ -0,0 +1,150 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright 2022 Open Networking Foundation
+
+package pfcpiface
+
+import (
+	"io/ioutil"
+	"testing"
+
+	//nolint:staticcheck // Ignore SA1019.
+	// Upgrading to google.golang.org/protobuf/proto is not a drop-in replacement,
+	// as also P4Runtime stubs are based on the deprecated proto.
+	"github.com/golang/protobuf/proto"
+	p4ConfigV1 "github.com/p4lang/p4runtime/go/p4/config/v1"
+)
+
+//nolint:unused
+func setupNewTranslator() *P4rtTranslator {
+	p4infoBytes, _ := ioutil.ReadFile(p4InfoPath)
```

Comment:
I would prefer to use another const variable for `p4InfoPath`. Actually, this shouldn't work because `p4InfoPath` points to the container's filesystem (/bin).

Reply:
defined a new path as const. It works with both `make test` and `go test`.